src/axolotl/utils/collators/batching.py

```
@@ -122,6 +127,28 @@ def __call__(self, features, return_tensors=None):
+                    prepare_fa_kwargs_from_position_ids,
+                )
+
+                (cu_q, cu_k), (max_q, max_k) = prepare_fa_kwargs_from_position_ids(
+                    features["position_ids"]
+                )
+                features["cu_seq_lens_q"] = cu_q
+                features["cu_seq_lens_k"] = cu_k
+                features["max_length_q"] = int(max_q)
+                features["max_length_k"] = int(max_k)
+            except (
+                Exception
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)